File: aula_12-04/0_calculadora.cpp
#include <iostream>
using namespace std;

int main() {
    double valor1, valor2;
    char operador;
    double resultado = 0;

    cout << "Digite o primeiro valor: ";
    cin >> valor1;
    cout << "Digite o operador: ";
    cin >> operador;
    cout << "Digite o segundo valor: ";
    cin >> valor2;

    if (operador == '+') {
        resultado = valor1 + valor2;
    } else {
        cout << "Operador inválido";
    }

    cout << "Resultado: " << resultado << endl;

    return 0;
}

File: aula_12-04/0_calculadora.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite o primeiro valor: ', (valor1) => {
  rl.question('Digite o operador: ', (operador) => {
    rl.question('Digite o segundo valor: ', (valor2) => {
      let resultado = 0;

      // Cálculo da operação

      console.log(`Resultado: ${resultado}`);
      rl.close();
    });
  });
});
